drivers/impedance-analyzer/ad5933/ad5933.c

```diff
@@ -44,10 +44,15 @@
 #include "ad5933.h"
 #include <math.h>
 
+#ifndef M_PI
+#define M_PI 3.14159265359
+#endif
+
 /******************************************************************************/
 /************************** Constants Definitions *****************************/
 /******************************************************************************/
 const int32_t pow_2_27 = 134217728ul;      // 2 to the power of 27
+const double radToDeg180ovPi = 180 / M_PI;
```

Comment:
Replace please camelCase with snake_case: `radToDeg180ovPi` -> `rad_to_deg_180ov_pi`

Reply:
Will change this